packages/content/src/enrichments.ts

```diff
@@ -573,20 +576,47 @@ export const enrichContent = (
   const defaults: Partial<IHubContent> = {
     errors: [],
   };
-  const contentWithPortalUrls = {
+  const contentWithDefaults = {
     ...defaults,
     ...content,
-    ...getPortalUrls(content, requestOptions),
   };
   // fetch any missing or requested enrichments
-  return fetchEnrichments(contentWithPortalUrls, requestOptions).then(
+  return fetchEnrichments(contentWithDefaults, requestOptions).then(
     (fetched) => {
-      // enrich dates now that we have metadata (if any)
-      const enriched = _enrichDates(fetched);
-      // if this is a layer (type: Feature Layer, Table, Raster Layer)
-      // or a feature or map service and caller has supplied a layer id
-      // we want the content to represent the layer instead of the service
-      return getLayerContent(enriched, requestOptions.layerId) || enriched;
+      const { layerId } = requestOptions;
+      const {
+        item,
+        slug,
+        errors,
+        data,
+        metadata,
+        groupIds,
+        orgId,
+        // TODO: org, ownerUser
+        server,
+        layers,
+        // TODO: these should be fetched instead of pulled from the dataset
+        recordCount,
+        boundary,
+        statistics,
+      } = fetched;
+      // org enrichment is not yet implemented
+      const org = orgId && { id: orgId };
+      return composeContent(item, {
+        layerId,
+        slug,
+        data,
+        errors,
+        metadata,
+        groupIds,
+        org,
+        server,
+        layers,
+        recordCount,
+        boundary,
+        statistics,
+        requestOptions,
+      });
```

Comment:
more payoff, `enrichContent()` is now using the _exact_ same logic as `datasetToContent()`, this is exactly what we're going to do in the new `fetchContent()` fn that will prioritize fetching fresh enrichments from portal instead over fetching from the Hub API.